CompuRacer_Core/src/gui.py

```diff
@@ -0,0 +1,424 @@
+import json
+import os
+import sys
+
+from PyQt5.QtCore import Qt, QTimer
+from PyQt5.QtWidgets import QPushButton, QSystemTrayIcon, QMenu, QAction, QMainWindow, QVBoxLayout, QLabel, QTableWidget, QTableWidgetItem, QTabWidget, QWidget, QMessageBox, QLineEdit, QHBoxLayout, QApplication
+from PyQt5.QtGui import QIcon
+
+
+def load_json_batches(directory) -> [str]:
```

Comment:
Mss passender: load_json_batches_names

Reply:
De filename is trouwens altijd gelijk aan de batch name, dus je hoeft de files niet per se in te lezen..